python/lsst/afw/table/base/baseContinued.py

```diff
@@ -21,15 +21,15 @@
 #
 from __future__ import absolute_import, division, print_function
 
-__all__ = ["Catalog"]
-
 from future.utils import with_metaclass
 
 import numpy as np
 
 from lsst.utils import continueClass, TemplateMeta
 from .base import BaseRecord, BaseCatalog
 
+__all__ = ["Catalog"]
+
```

Comment:
It was announced: https://community.lsst.org/t/--all---should-go-before-imports-in-python-code/1769/3